DCCpp_Uno/src/PacketRegister.cpp

```diff
@@ -225,9 +225,10 @@ void RegisterList::readCV(const char *s) volatile{
 
     bRead[2]=0xE8+i;
 
-    loadPacket(0,resetPacket,2,3);          // NMRA recommends starting with 3 reset packets
-    loadPacket(0,bRead,3,5);                // NMRA recommends 5 verfy packets
-    loadPacket(0,resetPacket,2,1);          // forces code to wait until all repeats of bRead are completed (and decoder begins to respond)
+loadPacket(0,resetPacket,2,3); // NMRA recommends starting with 3 reset packets
+loadPacket(0,bRead,3,5); // NMRA recommends 5 verify packets
+loadPacket(0,bRead,3,6); // NMRA recommends 6 write or reset packets for decoder recovery time
```

Comment:
swap this for ```loadPacket(0, idlePacket, 2, 6);```. The decoder will be processing the bRead packet and sending the ack responses at this point and the idle packet will be ignored by the decoder.

The code comment needs to be updated as well.